calling-apis/problem-set-requests-with-js.md

```diff
@@ -0,0 +1,296 @@
+# Problem Set: Requests With JS
+
+## Directions
+
+Read the problem prompt, which will ask you to write JavaScript code to find certain answers.
+
+Once you found the answers, input them into the answers prompt below.
+
+## Preparation: LocationIQ Forward Geocoding API
+
+Use the [LocationIQ's Forward Geocoding API](https://locationiq.com/geocoding). Before starting this problem prompt:
+
+1. Obtain an API key for LocationIQ's Forward Geocoding API
+1. Familiarize yourself with the LocationIQ API Documentation
+1. Familiarize yourself with the `axios` library
+1. Open Postman for manual testing
+
+![Screenshot of LocationIQ's feature on the Geocoding API](../assets/calling-apis_problem-set-requests-with-js_location-iq-geocoding.png)
+
+## Problem Prompt
+
+Create a JavaScript program that finds the latitude and longitude coordinates for the [seven wonders of the New7Wonders of the World](https://en.wikipedia.org/wiki/New7Wonders_of_the_World), which are the following:
+
+| Wonder              |
+| ------------------- |
+| Great Wall of China |
+| Petra               |
+| Colosseum           |
+| Chichen Itza        |
+| Machu Picchu        |
+| Taj Mahal           |
+| Christ the Redeemer |
+
```

Comment:
provide a suggestion about how to approach this. I'm extrapolating from your comments in the instructor notes and explicitly adding the recommendation that students start simply with a single request, with the rate limit reminder up here so they don't accidentally lock their key if they get started without scrolling all the way down to the current warning.
```suggestion

Remember that many API providers will deactivate an API key if they detect too many requests within a short span of time. Since adding pauses or timeouts to JavaScript code can be tricky, it may be worth starting with a program that retrieves the data for a single location, and attempt the Extra Challenges after that is working.

```